examples/04_scalability.py

```diff
@@ -147,14 +151,13 @@ def wrapped_func(*args, **kwargs):
 # Check now that prediction is still as good
 model = pipeline.make_pipeline(column_trans, log_ref)
```

Comment:
Unrelated to your PR, but I just realized that there is a typo in the whole example: "log_ref" should be changed to "log_reg".